python/shotgun_desktop/initialization/install.py

```diff
@@ -37,7 +37,7 @@ class InstallThread(QtCore.QThread):
     def __init__(self):
         QtCore.QThread.__init__(self)
         self._auth = None
-        self._logger = logging.getLogger("tk-framework-desktopstartup.initialization.install")
+        self._logger = logging.getLogger("tk-desktop.initialization.install")
```

Comment:
This simple mistake prevents us from knowing what broke during when a client has an issue setting up a core when Toolkit is initialized. Such errors will now be reported in the logs.